src/actions/get-btc-finality-by-block-hash.ts

```diff
@@ -0,0 +1,26 @@
+import { type Chain, type Hash, type Client, type Transport } from "viem"
+
+export async function getBtcFinalityByBlockHash<
+  TTransport extends Transport = Transport,
+  TChain extends Chain | undefined = Chain | undefined
+>(client: Client<TTransport, TChain>, parameters: { hash: Hash }) {
+  const { hash } = parameters
+  try {
+    const response = await client.transport.request({
```

Comment:
This call has to be sent to the op-node, not the default op-geth client. That's why the original code gets a new transport from the given client with this code:

```js
    const opNodeHttp = http(client.chain.rpcUrls.opNode.http[0])
    const transport = opNodeHttp({})
```

Reply:
oops! Makes sense ! Rollbacked here [90644c6](https://github.com/hemilabs/hemi-viem/pull/4/commits/90644c66ca8de42caef2971603f8db0ce67de8f6)